tests/functional/glusterd/test_glusterd_start_stop_output.py

```diff
@@ -0,0 +1,86 @@
+#  Copyright (C) 2021 Red Hat, Inc. <http://www.redhat.com>
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+"""Test Description:
+    Consistent info display while starting and stopping glusterd.
+"""
+
+from time import sleep
+from glusto.core import Glusto as g
+from glustolibs.gluster.gluster_base_class import GlusterBaseClass, runs_on
+from glustolibs.gluster.exceptions import ExecutionError
+from glustolibs.gluster.gluster_init import (restart_glusterd,
+                                             wait_for_glusterd_to_start)
+
+
+@runs_on([['replicated', 'distributed', 'arbiter', 'dispersed',
+           'distributed-replicated', 'distributed-arbiter',
+           'distributed-dispersed'], ['glusterfs']])
+class TestGlusterdStartStopOutput(GlusterBaseClass):
+    def tearDown(self):
+        # Restart glusterd on nodes for which it was stopped
```

Comment:
Glusterd is not stopped in these servers[3:5] in the test.

Reply:
@balakreddy Yes, in this TC, I'm dealing with only one server. Does that need to be modified to cover all servers ? From the TC description in Polarion I took it that it is to be done in only one server. But I guess it can be done in a manner to be performed on all servers. Would that be required ?

Reply:
@srijan-sivakumar Performing stop and start glusterd on one server would be fine.

No need to perform on all servers. you can remove the restart glusterd section in the teardown.

Reply:
Ack.